.github/workflows/run-danger.yml

```diff
@@ -6,6 +6,6 @@ on:
 
 jobs:
   dangermattic:
-    uses: Automattic/dangermattic/.github/workflows/reusable-run-danger.yml@trunk
+    uses: Automattic/dangermattic/.github/workflows/reusable-run-danger.yml@v1.0.0
```

Comment:
@leandroalonso: we can use this PR to add what you mentioned [here](https://github.com/Automattic/pocket-casts-ios/pull/1369#issuecomment-1910063436): skipping Danger on drafts, either completely or on some of the checks.

I think a Draft PR can also serve as a way to make sure CI will succeed, so my first instinct is to keep most checks there. But this is totally up to the teams and their own process, so let me know what you think makes more sense 🙂

Reply:
> we can use this PR to add what you mentioned https://github.com/Automattic/pocket-casts-ios/pull/1369#issuecomment-1910063436: skipping Danger on drafts, either completely or on some of the checks.

Can we only skip the checks like labels, milestones, etc? The type of thing you are not worried about until the PR is ready. I think the rest is perfectly fine to leave!

Reply:
I've made some changes here 4ccc086ee943df5cf30d0db5f53ec1f375d46248, avoiding some checks and also to prevent the job to run unnecessarily while the PR is a Draft.